src/app-layer-ftp.c

```diff
@@ -189,6 +189,8 @@ static int FTPParseRequestCommand(void *ftp_state, uint8_t *input,
     if (input_len >= 4) {
         if (SCMemcmpLowercase("port", input, 4) == 0) {
             fstate->command = FTP_COMMAND_PORT;
+        } else if (SCMemcmpLowercase("auth tls", input, 8) == 0) {
```

Comment:
What about using input_len as an argument to SCMemcmpLowercase instead? E.g.:
```c
} else if (SCMemcmpLowercase("auth tls", input, input_len) == 0) {
```
Or would that be stupid?